components/icon/add_icon.jsx

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React from 'react';
+
+import {FormattedMessage} from 'react-intl';
+
+export default class AddIcon extends React.PureComponent {
+    render() {
+        return (
+            <FormattedMessage
+                id='generic_icons.add'
+                defaultMessage='Add Icon'
+            >
+                {(title) => (
+                    <i
+                        className='fa fa-plus'
+                        title={title}
+                    />
+                )}
+            </FormattedMessage>
+        );
+    }
+}
```

Comment:
I don't think we need to add any tests fpr these since they just render a constant value, so there's nothing particularly valuable to test there. This PR is already really large as it is, and tests would just add to that.